app/client/src/types.ts

```diff
@@ -1,11 +1,31 @@
+export interface MyObject {
```

Comment:
This is equivalent to `Record<string, string>` right? So no need to define an interface, can just use a type alias. Could also make it generic, and use a clearer name, I'd suggest:
```
type Dict<T> = Record<string, T>
```